bigframes/geopandas/__init__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import geopandas as gpd
+
+
+class GeoSeries(gpd.GeoSeries):
+    def distance(self):
```

Comment:
As we discussed in chat, let's do `x` and `y` properties to start to avoid complexities with binary operators / alignment.

Also, let's do a `warnings.warn` with a new `bigframes.exceptions.PreviewWarning` and add preview info to the docstrings to indicate that these features may change until we release them fully.

Reply:
Resolved on new PR today.